solidity/dashboard/src/components/glossary/Navigation.jsx

```diff
@@ -31,6 +31,7 @@ const GlossaryLink = ({ label, to }) => {
   return (
     <li>
       <Link
+        containerId="main-content-wrapper"
```

Comment:
Before I forget: we may want to check with @liz-shinn, but it feels like the side navigation used for the glossary (now resources) could just become the same top tab navigation that we're using for other subnavigation?

Reply:
Hadn't considered this 🤔 I'm not sure, tbh... 

I like the structure of the side navigation in this resources (glossary) as it stands for a couple reasons:
- it mirrors a common pattern for docs, so it looks at a glance like this is a doc section
- it allows for all of this content to be in one place (the diagram doesn't really warrant it's own page imo)

But I *do* like the idea of rearranging it to match the other pages for a few other reasons:
- it's consistent with other pages treatment of subnavigation, so teaches user to look in the same place
- it allows for more horizontal screen real estate, especially for diagrams (I can see there being a bunch of info viz we could include here)
- it's more scalable and better suited to adding more items, in particular links out to docs

So actually, TL;DR I've talked myself into doing what @Shadowfiend suggests here, and agreed that it makes sense to make the side nav for glossary the same top tab navigation that we're using for other subnavigation. 👍 💯

Reply:
so I'll remove the right side menu and use sub nav on the top 👍

Reply:
Updating the Figma file to reflect this change to the Resources page 💯